xml2json/index.d.ts

```diff
@@ -0,0 +1,67 @@
+// Type definitions for xml2json 0.10
+// Project: https://github.com/buglabs/node-xml2json
+// Definitions by: Dolan Miu <https://github.com/dolanmiu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+export function toJson(xml: string, options?: JsonOptions): Object;
```

Comment:
This may return a `string`: `node -p 'typeof require("xml2json").toJson( "<foo>bar</foo>")'`

You might want overloads:
```ts
export function toJson(xml: string, options?: { object?: false } & JsonOptions): string;
export function toJson(xml: string, options?: { object: true } & JsonOptions): {};
```

Then remove the option from `JsonOptions`.